client/src/components/Users/ListUsers.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { Fragment, useEffect, useState } from "react";
+import { Fragment} from "react";
```

Comment:
Just to note, you can use `<></>` instead of `<Fragment></Fragment>` if you want, they should compile to the same.